var/spack/repos/builtin/packages/root/package.py

```diff
@@ -211,11 +211,15 @@ class Root(CMakePackage):
     depends_on("libsm", when="+x")
 
     # OpenGL
-    depends_on("ftgl@2.4.0:", when="+x+opengl")
-    depends_on("glew", when="+x+opengl")
-    depends_on("gl", when="+x+opengl")
-    depends_on("glu", when="+x+opengl")
-    depends_on("gl2ps", when="+x+opengl")
+    depends_on("ftgl@2.4.0:", when="+opengl")
+    depends_on("glew", when="+opengl")
+    depends_on("gl2ps", when="+opengl")
+    if sys.platform != "darwin":
+        depends_on("gl", when="+x+opengl")
+        depends_on("glu", when="+x+opengl")
+    else:
+        depends_on("gl", when="+opengl")
+        depends_on("glu", when="+opengl")
```

Comment:
Given that `conflicts("+opengl", when="~x")` if `sys.platform != darwin`, I think this is equivalent to the simpler...

```suggestion
    depends_on("gl", when="+opengl")
    depends_on("glu", when="+opengl")
```